spring-graphql/src/main/java/org/springframework/graphql/execution/DelegatingSubscriptionExceptionResolver.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2002-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.graphql.execution;
+
+import graphql.ErrorType;
+import graphql.GraphQLError;
+import graphql.GraphqlErrorBuilder;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.springframework.util.Assert;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * An implementation of {@link SubscriptionExceptionResolver} that is trying to map exception to GraphQL error
+ * using provided implementation of {@link SubscriptionExceptionResolver}.
+ * <br/>
+ * If none of provided implementations resolve exception to error or if any of implementation throw an exception,
+ * this {@link SubscriptionExceptionResolver} will return a default error.
+ *
+ * @author Mykyta Ivchenko
+ * @see SubscriptionExceptionResolver
+ */
+public class DelegatingSubscriptionExceptionResolver implements SubscriptionExceptionResolver {
+    private static final Log logger = LogFactory.getLog(DelegatingSubscriptionExceptionResolver.class);
+    private final List<SubscriptionExceptionResolver> resolvers;
+    private final List<GraphQLError> defaultErrors;
```

Comment:
I'm not sure about this list of fixed, static errors, which is not even configurable from anywhere. An application can add any number of resolvers, which means it can add a very general one at the end to deal with any Throwable. I would remove this for now.